examples/wfs.js

```diff
@@ -0,0 +1,150 @@
+/* global itowns, document, renderer, Uint8Array */
+// # Planar (EPSG:3946) viewer
+
+var extent;
+var viewerDiv;
+var view;
+var flyControls;
+
+// Define projection that we will use (taken from https://epsg.io/3946, Proj4js section)
+itowns.proj4.defs('EPSG:3946',
+    '+proj=lcc +lat_1=45.25 +lat_2=46.75 +lat_0=46 +lon_0=3 +x_0=1700000 +y_0=5200000 +ellps=GRS80 +towgs84=0,0,0,0,0,0,0 +units=m +no_defs');
+
+// Define geographic extent: CRS, min/max X, min/max Y
+extent = new itowns.Extent(
+    'EPSG:3946',
+    1837816.94334, 1847692.32501,
+    5170036.4587, 5178412.82698);
+
+// `viewerDiv` will contain iTowns' rendering area (`<canvas>`)
+viewerDiv = document.getElementById('viewerDiv');
+
+// Instanciate PlanarView*
+view = new itowns.PlanarView(viewerDiv, extent, { renderer: renderer });
+view.tileLayer.disableSkirt = true;
+
+// instanciate controls
+flyControls = new itowns.FlyControls(view, { focusOnClick: true });
+flyControls.moveSpeed = 1000;
+
+// Add an WMS imagery layer (see WMS_Provider* for valid options)
+view.addLayer({
+    url: 'https://download.data.grandlyon.com/wms/grandlyon',
+    networkOptions: { crossOrigin: 'anonymous' },
+    type: 'color',
+    protocol: 'wms',
+    version: '1.3.0',
+    id: 'wms_imagery',
+    name: 'Ortho2009_vue_ensemble_16cm_CC46',
+    projection: 'EPSG:3946',
+    transparent: false,
+    extent: extent,
+    axisOrder: 'wsen',
+    options: {
+        mimetype: 'image/jpeg',
+    },
+});
+
+view.camera.camera3D.position.set(1839739, 5171618, 910);
+view.camera.camera3D.lookAt(new itowns.THREE.Vector3(1840839, 5172718, 0));
+
+// Request redraw
+view.notifyChange(true);
+
+function colorFunctionLine(layer, node, feature) {
+    feature.traverse(function _(n) {
+        var rgb;
+        if (n.material) {
+            if (n.properties) {
```

Comment:
They are copied in the returned Mesh: either as mesh.properties (for a single feature) or in mesh.features (for a feature collection)